components/header/index.tsx

```diff
@@ -1,5 +1,8 @@
+/* eslint-disable react/jsx-sort-props */
```

Comment:
Can this be removed if the props for MegaMenuBuilder are sorted alphabetically?

Reply:
Yep, I didn't intentionally add that one. Removing.